src/enclave/connection.ts

```diff
@@ -312,7 +350,22 @@ export class Enclave implements EnclaveWriter {
 	}
 
 	private onOpen(_: Event) {
+		const calls = [];
```

Comment:
You can also add a testcase in `src/enclave/connection.spec.ts`. There you have access to the `EnclaveMockProvider` which allows you to call `close` and `open` which in turn _should_ give you the ability to test  this path with ease.